.docs/Get-VSTeamTaskGroup.md

```diff
@@ -0,0 +1,87 @@
+<!-- #include "./common/header.md" -->
+
+# Get-VSTeamTaskGroup
+
+## SYNOPSIS
+
+<!-- #include "./synopsis/Get-VSTeamTaskGroup.md" -->
+
+## SYNTAX
+
+## DESCRIPTION
+
+<!-- #include "./synopsis/Get-VSTeamTaskGroup.md" -->
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+
+```powershell
+
+$methodParameters = @{
+   ProjectName = "some_project_name"
+}
+
+Get-VSTeamTaskGroup @methodParameters
+```
+
+### -------------------------- EXAMPLE 2 --------------------------
+
+```powershell
+
+$methodParameters = @{
+   ProjectName = "some_project_name"
+   Id          = "Task_group_id"
+}
+
+Get-VSTeamTaskGroup @methodParameters
+```
+
+### -------------------------- EXAMPLE 3 --------------------------
+
+```powershell
+
+$methodParameters = @{
+   ProjectName = "some_project_name"
+   Name        = "Task_group_name"
+}
+
+Get-VSTeamTaskGroup @methodParameters
+```
+
+## PARAMETERS
```

Comment:
Make sure you add the YAML section under each parameter to define the type, required, parameter set, accepts pipeline, etc.